docs: improve documentation to help user get started (#126)
puppeteer/replay: README.md

###### [API](https://github.com/puppeteer/replay/blob/main/docs/api) | [Contributing](https://github.com/puppeteer/replay/blob/main/docs/contributing.md)

> > Puppeteer Replay is a library which provides an API to replay and stringify recordings created using [Chrome DevTools Recorder](https://developer.chrome.com/docs/devtools/recorder/)

Reviewer: See https://www.dictionary.com/e/that-vs-which/

```
npm install puppeteer --save
```
## Getting started with Puppeteer Replay

You can use Puppeteer Replay to:

1. **Replay recording**. Replay recording with CLI or using [the replay lib API](/examples/replay-from-file-using-puppeteer/main.js).

> 2. **Customize replay**. Customize how a recording is run (e.g. capture screenshots after each step, intergrate with 3rd party libraries).

Reviewer: ```suggestion
2. **Customize replay**. Customize how a recording is run. For example, capture screenshots after each step or integrate with 3rd party libraries.
```

> 3. **Transform recoding**. Customize how a recording is stringified. (e.g. transform the recording into other format like [Cypress test script](https://github.com/cypress-io/cypress-chrome-recorder))

Reviewer: ```suggestion
3. **Transform recoding**. Customize how a recording is stringified. For example, transform the recording into another format, like [Cypress test script](https://github.com/cypress-io/cypress-chrome-recorder))
```

## 1. Replay recording

> Download this [example recording](https://storage.googleapis.com/web-dev-uploads/file/dPDCek3EhZgLQPGtEG3y0fTn4v82/vzQbv2rUfTz2DEmx06Gv.json).

Reviewer 2: should we say to save it as `recording.js`?

> Download this [example recording](https://storage.googleapis.com/web-dev-uploads/file/dPDCek3EhZgLQPGtEG3y0fTn4v82/vzQbv2rUfTz2DEmx06Gv.json).

Author: ```suggestion
Download this [example recording](https://storage.googleapis.com/web-dev-uploads/file/dPDCek3EhZgLQPGtEG3y0fTn4v82/vzQbv2rUfTz2DEmx06Gv.json) and save it as `recording.json`.
```

await runner.run();
```

## 2. Customize replay

> The library offers way to customize how a recording is run. You can extend

Reviewer: ```suggestion
The library offers a way to customize how a recording is run. You can extend
```
